docs/tutorial/notifications.md

```diff
@@ -1,30 +1,100 @@
 # Notifications (Windows, Linux, macOS)
 
+## Overview
+
 All three operating systems provide means for applications to send notifications
 to the user. Electron conveniently allows developers to send notifications with
 the [HTML5 Notification API](https://notifications.spec.whatwg.org/), using
 the currently running operating system's native notification APIs to display it.
 
-**Note:** Since this is an HTML5 API it is only available in the renderer process. If
-you want to show Notifications in the main process please check out the
-[Notification](../api/notification.md) module.
+## Example
+
+### Show notifications in the Renderer process
```

Comment:
The part above here:

> Electron conveniently allows developers to send notifications with the [HTML5 Notification API](https://notifications.spec.whatwg.org/), using the currently running operating system's native notification APIs to display it

should be listed under the `Renderer process` part. The overview should also mention in brief that the technique for renderer and main processes is different before jumping into each example.